deploy/ocp-templates/glusterfs-template.yaml

```diff
@@ -35,6 +35,12 @@ objects:
         - image: gluster/gluster-centos:latest
           imagePullPolicy: IfNotPresent
           name: glusterfs
+          QoS Tier:
+            memory: Guaranteed
```

Comment:
This needs to be Burstable for now due to the current state of QoS tiers in Kubernetes.  Per @derekwaynecarr comment in the BZ:

```
@jerms, i might be inclined to have the QoS class Burstable where the cpu
request is specified, but the limit is not.  A memory limit should be
specified.  That feels like the sweet spot today given the current state of the
product, WDYT?```